build-release

```diff
@@ -406,6 +407,26 @@ build () {
 		engine_archive_basename="${target}"
 		cmake_opts="${cmake_opts} -DUSE_LTO=ON -DBUILD_CLIENT=ON -DBUILD_SERVER=ON -DBUILD_TTY_CLIENT=ON -DBUILD_GAME_NACL=OFF -DBUILD_GAME_NACL_NEXE=OFF -DBUILD_CGAME=OFF -DBUILD_SGAME=OFF"
 
+		local strip=''
+		if "${system_windows}" && ! "${host_windows}"
+		then
+			if "${arch_i686}"
+			then
+				bitness='32'
```

Comment:
Maybe I'm the one who wrote that code and forgot to use `local`, but maybe we can use `local` on those?